tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/debug_observer.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file debug_observer.h
+ * @brief Debug Taskflow Observer
+ *
+ * @author Matthew Powelson
+ * @date July 15. 2020
```

Comment:
Needs updating

Reply:
I copied this from the you created so kept the same date.

Reply:
Oh ok. Got it.